### ISSUE TITLE
Build failures on Windows platforms (v3.2.0)

### ISSUE THREAD

**Issue by marekgutten** [alicevision/AliceVision#1670]
The latest release v3.2.0 has compilation problems, at least on Windows. For example, vcpkg fails:

![image](https://github.com/alicevision/AliceVision/assets/80094850/b728ab5b-8444-4e9f-9d42-a703ad7446e8)

Manually updating Jasper to version 4.2.0 solved my problem.
Can you please update jasper in the vcpkg project?

**Comment by cbentejac** [comment]
Hi @marekgutten, thanks for reporting the issue. 

Just to be sure we're on the same page: you got this error by cloning https://github.com/alicevision/vcpkg and using the `vcpkg install` command described in https://github.com/alicevision/AliceVision/blob/develop/INSTALL.md from the `alicevision_master` branch?

I'm asking because I tried to do a clean build from scratch in these conditions and did not run into that error for Jasper (I did not run into any error, as a matter of fact), so I'm trying to understand whether there is something we did differently, or if there is a cache or something I forgot to clean properly before rebuilding everything.
